selfdrive/car/docs_definitions.py

```diff
@@ -50,6 +50,7 @@ class CarInfo:
   min_steer_speed: Optional[float] = None
   min_enable_speed: Optional[float] = None
   good_torque: bool = False
+  harness: Optional[Enum] = None
```

Comment:
don't think you want to set a default here. it shouldn't be easy to forget to specify the harness. defaults make sense for the make subclasses though

Reply:
for the makes that specify a default package, we'd need to specify the default harness to also be None, like in Hyundai and Honda, but I can do this. defeats the purpose in those two files